src/run_v2g.py

```diff
@@ -0,0 +1,89 @@
+"""Step to aggregate variant-to-gene assesments."""
+from __future__ import annotations
+
+from functools import reduce
+from typing import TYPE_CHECKING
+
+import hydra
+import pyspark.sql.functions as f
+
+from etl.json import validate_df_schema
+
+if TYPE_CHECKING:
+    from omegaconf import DictConfig
+
+from etl.common.ETLSession import ETLSession
+from etl.v2g.functional_predictions.vep import main as extract_v2g_from_vep
```

Comment:
does this sound like a good idea?

Reply:
renaming the main function you mean?
Why I did it like this: since each module features the functions needed to build each dataset I thought it was a good idea to have a `main` function to make clear what is the main function that hosts all the logic. And when I import that in the ETL process, I want to give it a specific name so that `run_v2g` is easy to follow.

I don't know if this is the best practice, so I am open to changing it.